src/manifest/dbtProject.ts

```diff
@@ -923,14 +923,14 @@ select * from renamed
     return crypto.createHash("md5").update(projectRoot).digest("hex");
   }
 
-  private async findModelInTargetfolder(modelPath: Uri, type: string) {
+  private async findModelInTargetfolder(modelPath: Uri) {
     const targetPath = this.getTargetPath();
     if (!targetPath) {
       return;
     }
     const baseName = path.basename(modelPath.fsPath);
     const targetModels = await workspace.findFiles(
-      new RelativePattern(targetPath, `${type}/**/${baseName}`),
+      new RelativePattern(targetPath, `compiled/**/${baseName}`),
```

Comment:
here we should find the relative path of the model and use the relative path for finding inside the compiled/run directory